### ISSUE TITLE
Allow choosing between intervalically equivalent note indicators

### ISSUE THREAD

**Issue by cryoranger** [neanes/neanes#833]
**This bug was found in**

- [x] Desktop
- [ ] Web App
- [ ] Both

**Describe the bug**
I want to change the note for the Ga soft chromatic fthora to Ni and I am not allowed.

**To Reproduce**
Add the soft chromatic Ga fthora and try to change the Fthora note to Ni.

**Expected behavior**
I do not have a theory book at hand with the respective theoretical background for screenshots, but the Ga soft chromatic fthora is used for Ga, Ke, Ni'.

**Screenshots or files**
I have found this in a lesson that appears in two books.

From Μουσική Πανδέκτης Γ' "Μαθηματάριον" (1851):
![image](https://github.com/user-attachments/assets/0e9a8b25-1d07-42b5-a53e-16b791ee478f)

From "Μαθηματάριον" (2017):
![image](https://github.com/user-attachments/assets/17f622a7-a5ca-4786-be3e-2151fafc454a)

**Desktop Version (please complete the following information, if applicable):**

- OS: Windows 10
- Version 0.4.47

**Comment by basil** [comment]
Thank you for posting this example from "Of Old the Prophets" by Koukouzelis on page 10 of the 1851 _Mathimatarion_. I was able to typeset this in Neanes using a **Fthora Note** of Ga (as expected), but when I clicked the **Note Indicator** button a Ga was placed on the note rather than a Ni:

![selection](https://github.com/user-attachments/assets/86cb032c-54ee-4157-9379-2f87253f0311)

This is not _entirely_ wrong, as the Ga and Ni intervallically equivalent here, and indeed playback is correct. But I think there is a missing feature. Both a Ga and Ni would be acceptable **Note Indicators** here, as they are intervallically equivalent, so the choice of which one to display is a cosmetic choice made by the editor. And here I agree with the editor that Ni is more readable, as this piece is borrowing a Second Mode phrase which is physically on Ni in that mode and thus more easily recognizable with a Ni **Note Indicator.**

So while both a Ga and Ni **Note Indicator** are intervallically equivalent, a Ni **Note Indicator** would be more readable here as well as matching the classical publication. I think the software should support both options ideally.

I'm not sure how this should be exposed in the user interface. The user interface for these two features is already confusing enough as it is, as described in #664.
